src/Client.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Keboola\AzureEventGridClient;
+
+use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+
+class Client
+{
+    private const API_VERSION = '2018-01-01';
```

Comment:
To bude asi copy&paste tak to upravit na aktualne datum

Reply:
To je verze Azure API viz. https://docs.microsoft.com/en-us/rest/api/eventgrid/dataplane/publishevents/publishevents